Makefile

```diff
@@ -77,6 +90,7 @@ test-e2e-with-kafka-testonly:
 
 test-e2e-with-kafka:
 	UNINSTALL_STRIMZI="false" ./hack/strimzi.sh
+	./hack/tracing.sh
```

Comment:
I think a lot of the rules in this file could be optimized:
test-e2e-with-kafke: uninstall-strimzi uninstall-tracing-opentelemetry 
There is a catch with dependencies ordering and the fact that they are only built once but are not we use the targets once per time?

Reply:
cool thanks for the history lesson :)